spec/registry.md

```diff
@@ -1109,6 +1109,31 @@ the functions `:datetime` and `:time`:
   - `true`
   - `false`
 
+The following _option_ and its values will be REQUIRED to be available on
+the functions `:datetime`, `:date`, and `:time`.
+This option currently has a Maturity Level of **Proposed**.
+
+- `timezone`
+  - A valid time zone identifier
+    (see [BCP175](https://www.rfc-editor.org/bpc/bpc175))
```

Comment:
This link is broken (should probably be https://www.rfc-editor.org/info/bcp175), but looking at the linked text (which in #911 I'd just copied from the previous), it states this:
> The normative reference for [time zone names] is the TZ Database [TZDB].

where the onward reference is to
> Eggert, P. and A. Olson, "Sources for Time Zone and Daylight Saving Time Data", 1987, <ftp://ftp.iana.org/tz/code/tz-link.htm>.

As accessing FTP links isn't as easy as it once was, I may only presume that https://data.iana.org/time-zones/tz-link.html is a valid current corresponding reference, but it does not establish anything clear about how time zone names are defined.

For comparison, the JS reference is more current and much more clear in its definition of the appropriate authority (IANA), and rather exactly defines how to look up the available names: https://tc39.es/ecma402/#sec-use-of-iana-time-zone-database

Reply:
The link has a typo (it should s/bpc/bcp/). We could link the TZDB directly and that's probably better than the BCP. As Mark pointed out, the real reference is buried inside the datafiles.

The JS reference is a good one and describes a similar workaround for IANA's cross-border zone merging scheme to that of others.

As mentioned elsewhere, I don't want to get into the business of defining all these things. We have the problem that CLDR/ICU "do stuff" with time zones and identifiers and that stuff, while not incompatible with what JS does, is more complex. 

I think what I'll do here is reference IANA's page and also LDML. In practice, we don't need to define "well-formed" or "valid" directly.